mainProject/services/userService.ts

```diff
@@ -0,0 +1,71 @@
+import { IUserRequestBody } from "../interfaces/user";
+import { v4 } from "uuid";
+import UserModel from "../models/user";
+import { Op } from "sequelize";
+
+class UserService {
+  async getAutoSuggestUsers(login: string, limit: number) {
+    const result = await UserModel.findAll({
+      where: {
+        is_deleted: false,
+        login: {
+          [Op.like]: `${login}%`,
```

Comment:
that will be case sensitive and don't allow to find a piece of login in the center and at the end of login

Reply:
This was intentional